snowpack/src/hmr-server-engine.ts

```diff
@@ -14,8 +14,13 @@ interface Dependency {
 export class EsmHmrEngine {
   clients: Set<WebSocket> = new Set();
   dependencyTree = new Map<string, Dependency>();
+  private liveReloadDelayMs: number = 0;
+  private currentBatch: object[] = [];
```

Comment:
totally fine as is, but if you wanted to go a step further in this PR or a follow up PR, it would be great to start typing these message objects in TypeScript. Basically something like:

```
type HMRMessage = {type: 'reload'} | {type: 'update', url: string};
```